### PR TITLE
Migrate to GenerateMethod's new return value

### PR DESCRIPTION
**Pull request by BladeMF**
Fix CS did change a lot of files for some reason.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,4 +21,6 @@
 __pycache__
 /doc/_ext/__pycache__
 .phpunit.result.cache
-.php_cs.cache
\ No newline at end of file
+.php_cs.cache
+
+/.vscode
```
Comment (dantleech) [inline .gitignore:26]: would be better to add this to your own `.gitignoreglobal` file in general.

**Comment by dantleech** [comment]
I push composer lock updates with a changelog normally, so I'll do this shortly.

**Comment by dantleech** [comment]
tests are failing...

File: tests/Unit/Extension/CodeTransformExtra/Rpc/GenerateMethodHandlerTest.php [hunk 2]
```diff
@@ -31,12 +36,17 @@ public function testProvidesOriginalSourceFromDiskIfPathIsNotTheGivenPath(): voi
...
+        $thisFileContents = file_get_contents(__FILE__);
 
         $this->generateMethod->generateMethod(
             $source,
             self::EXAMPLE_OFFSET
-        )->willReturn($transformedSource);
+        )->willReturn(new TextDocumentEdits(
+            TextDocumentUri::fromString(__FILE__),
+            TextEdits::fromTextEdits([
+                TextEdit::create(strlen($thisFileContents) - 1, 1, substr($thisFileContents, -1) ."1")
```
Comment (dantleech) [inline tests/Unit/Extension/CodeTransformExtra/Rpc/GenerateMethodHandlerTest.php:47]: why "1" ?

File: tests/Unit/Extension/CodeTransformExtra/Rpc/GenerateMethodHandlerTest.php [hunk 3]
```diff
@@ -31,12 +36,17 @@ public function testProvidesOriginalSourceFromDiskIfPathIsNotTheGivenPath(): voi
...
-        $transformedSource = SourceCode::fromStringAndPath(self::EXAMPLE_SOURCE, __FILE__);
+        $thisFileContents = file_get_contents(__FILE__);
 
         $this->generateMethod->generateMethod(
             $source,
             self::EXAMPLE_OFFSET
-        )->willReturn($transformedSource);
+        )->willReturn(new TextDocumentEdits(
+            TextDocumentUri::fromString(__FILE__),
+            TextEdits::fromTextEdits([
```
Comment (dantleech) [inline tests/Unit/Extension/CodeTransformExtra/Rpc/GenerateMethodHandlerTest.php:46]: can use `TextEdits::one(TextEdit..`

File: tests/Unit/Extension/CodeTransformExtra/Rpc/GenerateMethodHandlerTest.php (hunk 2 above)
Comment (BladeMF) [reply tests/Unit/Extension/CodeTransformExtra/Rpc/GenerateMethodHandlerTest.php:47]: I wanted the change to just add "1" to the file contents, but it didn't work if I just described a text edit at `strlen(...)`, so I replaced the last character with itself + "1".

**Comment by BladeMF** [comment]
> tests are failing...

I think I need to bump the version. Tried it.

**Comment by BladeMF** [comment]
I have no idea how to fix this... :-(

**Comment by dantleech** [comment]
The lock file needs to be updated (`composer update`)

**Comment by BladeMF** [comment]
Done, let's see if now it passes.

**Pull request merged**